user-guide/Standard_Apps/IDP/Using_DataMiner_IDP/Overview_of_the_IDP_app_UI.md

```diff
@@ -166,53 +166,50 @@ This tab displays an overview of the levels in the infrastructure, with KPIs and
 
 ## Processes
 
-This tab consists of the following subtabs:
-
-- **Schedules**: This subtab displays an overview of scheduled activities or processes, depending on whether IDP is used together with DataMiner Process Automation. If Process Automation is not used, the *New* button allows you to schedule a new discovery activity. With DataMiner Process Automation, the *New* button opens the Process Automation wizard.
-
-  In the Process Automation wizard, you can create or schedule processes consisting of one or more activities. To do so:
+This tab displays an overview of scheduled activities or processes, depending on whether IDP is used together with DataMiner Process Automation. If Process Automation is not used, the *New* button allows you to schedule a new discovery activity. With DataMiner Process Automation, the *New* button opens the Process Automation wizard.
```

Comment:
In case IDP is used together with DataMiner Process Automation,  this tab displays an overview of scheduled processes.